odk1-src/form-functions.rst

```diff
@@ -0,0 +1,674 @@
+.. spelling::
+
+  Anaranjado
+  Azul
+  Colores
+  Español
+  Púrpura
+  Qué
+  Rojo
+  Seleccione
+  arctan
+  arg
+  bday
+  bp
+  br
+  chkn
+  colores
+  fsh
+  gustan
+  jr
+  nodeset
+  prefs
+  seleccionados
+  te
+  tres
+  π
+  
+  
+	
+******************************
+Form Functions and Operators
+******************************
+
+:ref:`expressions` in :ref:`calculations`, :ref:`constraints <constraints>`, and :ref:`relevants <relevants>`
+can contain `XPath functions`_ and operators.
+
+.. _XPath functions: https://opendatakit.github.io/xforms-spec/#xpath-functions
+
+.. contents::
+  :local:   
+
+.. _functions:
+  
+Functions
+===========
+    
+.. _control-flow-functions:
+
+Control flow
+--------------
+
+.. function:: if(expression, then, else)
+
+  Returns :arg:`then` if :arg:`expression` evaluates to ``True``. 
+  Otherwise, returns :arg:`else`.
+
+  
+.. function:: not(arg)
+
+  Returns the opposite of :func:`boolean(arg) <boolean>`.
+
+  
+.. function:: coalesce(arg, arg) 	
```

Comment:
coalesce and boolean feel to my like utility functions. Or rather, they feel like functions that no one uses and they are front and center.